options/options_settable_test.cc

```diff
@@ -223,6 +223,8 @@ TEST_F(OptionsSettableTest, DBOptionsAllFieldsSettable) {
       {offsetof(struct DBOptions, wal_filter), sizeof(const WalFilter*)},
       {offsetof(struct DBOptions, file_checksum_gen_factory),
        sizeof(std::shared_ptr<FileChecksumGenFactory>)},
+      {offsetof(struct DBOptions, checksum_handoff_file_types),
```

Comment:
Why are you adding an option to the DBOptions that is not settable as an option?  You should be able to set this using the options code.